features/support/env.rb

```diff
@@ -1,5 +1,35 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__) + '/../../lib')
-require 'aruba/cucumber'
+
+# Has to be the first file required so that all other files show coverage information
+require 'simplecov'
```

Comment:
This is only necessary if we want coverage in the process the cucumber process and not the process under test, so it can be removed.  I think I made that change in the source project and didn't update this PR because the review process was stalled.  I only meant for `simplecov_setup` to be used in child process and not the cucumber process itself.